src/app/wallets/get-transactions-for-wallet.ts

```diff
@@ -60,7 +66,45 @@ export const getTransactionsForWallets = async ({
     const account = await accountRepo.findById(wallet.accountId)
     const depositFeeRatio =
       account instanceof Error ? (0 as DepositFeeRatio) : account.depositFeeRatio
-    walletDetailsByWalletId[wallet.id] = { currency: wallet.currency, depositFeeRatio }
+
+    const displayCurrency =
+      account instanceof Error
+        ? (DisplayCurrency.Usd as T)
+        : (account.displayCurrency as T)
+
+    let displayPriceRatioForPending:
+      | DisplayPriceRatio<"BTC", T>
+      | PriceServiceError
+      | undefined = await getCurrentPriceAsDisplayPriceRatio<T>({
+      currency: displayCurrency,
+    })
+    if (displayPriceRatioForPending instanceof Error) {
+      displayPriceRatioForPending = undefined
+    }
+
+    let walletPriceRatio = WalletPriceRatio({
+      usd: { amount: 1n, currency: WalletCurrency.Usd },
+      btc: { amount: 1n, currency: WalletCurrency.Btc },
+    })
```

Comment:
1 to 1? is this correct?

Reply:
Yea I need a way to make this clearer. This is meant to me a dummy value for the type checker and the real value we're interested in gets set on Ln 98 for non-btc currencies. It's passed on via `walletDetailsByWalletId` to `addPendingIncoming` and only gets used if wallet currency is not btc on Ln55 and Ln60 in tx-history.ts.

Reply:
I added a commit (https://github.com/GaloyMoney/galoy/pull/2434/commits/f37ff6b828084eea2f1187e3a666b284aa5a38b6) to make this a bit clearer, but this is enforced by convention and not type-checker still where the failure case is that the values affected are set to `0`.